playwright-e2e/tests/brain/brain-regression-test.spec.ts

```diff
@@ -0,0 +1,386 @@
+    await page.getByText('The goal of this project is to generate a large dataset that includes genomic, c').click();
+    await page.getByRole('button', { name: 'Who is conducting this research?' }).click();
+    await page.getByText('This project is being conducted by Count Me In. Our team is made up of cancer re').click();
+    await page.getByRole('button', { name: 'What are the genomic and molecular research aspects of this project?' }).click();
+    await page.getByText('This study involves genomic and molecular research to better define the genes an').click();
+    await page.getByRole('button', { name: 'Are there any costs for me to participate?' }).click();
+    await page.getByText('No, there are no costs to you. All associated costs related to acquiring medical').click();
+    await page.getByRole('button', { name: 'How has the brain tumor community been involved in the design of this project?' }).click();
+    await page.getByText('We have worked closely with patients, parents, and patient advocates within the ').click();
+    await page.getByRole('button', { name: 'If I am asked to share a tissue sample, will my tissue be used up?' }).click();
+    await page.getByText('If you do elect to share tissue as part of the study, we will take every measure').click();
+    await page.getByText('In order to get a sense of what archived samples are available for request, the ').click();
+    await page.getByRole('button', { name: 'Is this study a clinical trial?' }).click();
+    await page.getByText('This study is not a clinical trial. Participation will not have any impact on yo').click();
+    await page.getByRole('heading', { name: 'I/my child have been diagnosed with a brain tumor' }).click();
+    await page.getByRole('button', { name: 'How does this project work?' }).click();
+    await page.getByText('Click "Count Me In" and complete a simple online form to tell us about yourself/').click();
+    await page.getByText('If you are in the United States or Canada after you fill out the initial survey,').click();
+    await page.getByText('We\'ll also send you a simple kit to collect a saliva sample from you/your child,').click();
+    await page.getByRole('button', { name: 'What happens when my child with a brain tumor becomes an adult?' }).click();
+    await page.getByText('When your child reaches the age of consenting for themselves, they need to conse').click();
+    await page.getByText('If your child remains in the study when they reach the age of consenting for the').click();
+    await page.getByRole('button', { name: 'How can I spread the word about the Brain Tumor Project?' }).click();
+    await page.getByText('If you would like to learn more about how to spread the word about the project, ').click();
+    await page.getByRole('button', { name: 'How can I receive updates on the Brain Tumor Project?' }).click();
+    await page.getByText('If you would like updates on the project, please sign up for our mailing list.').click();
```

Comment:
thorough

Reply:
Only captures a few of the emails, but wanted to show how you could probe different parts of the same email without having to copy paste the entire text blob.